### PR TITLE
Case Study 1.

### PR DESCRIPTION
**Pull request by trojanekga**

### CODE REVIEW

**Comment by trojanekga** [comment]
Minor fixes.

File: index.html [hunk 1]
```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html>
```
Comment (mbMosman) [inline index.html]: You should have the attribute here on the html tag to indicate the language.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+            12010 Pacific Trails Road<br>
+            Zephyr, CA 95555
+            <br>
+            <br>
+            888-555-5555</p>
+            </div>
+        
+        
+        </main>
+       <footer>&copy; 2016 Pacific Trails Resort<br>
```
Comment (mbMosman) [inline index.html]: You should also have the word Copyright here before the copyright symbol.

File: yurts.html [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+            <h3>What is a yurt?</h3> 
+            <p>Our luxury yurts are permanent structures four feet off the ground. Each yurt has canvas walls, a wooden floor, and a roof dome that can be opened.</p>
+            <h3>How are the yurts furnished?</h3>
+            <p>Each yurt is furnished with a queen-size bed with down quilt and gas-fired stove. The luxury camping experience also includes electricity and a sink with hot and cold running water. Shower and restroom facilities are located in the lodge.</p>
+            <h3>What should I bring?</h3>
+            <p>Bring a sense of adventure and some time to relax! Most guests also pack comfortable walking shoes and plan to dress for changing weather with layers of clothing.</p>
+        
+        
+        </main>
+       <footer>&copy; 2016 Pacific Trails Resort<br>
```
Comment (mbMosman) [inline yurts.html]: You should also have the word Copyright here before the copyright symbol.

**Pull request closed without merging**